cmd/top-api-spinner.go

```diff
@@ -153,16 +162,20 @@ func (m *traceUI) View() string {
 			traceSt.addAPIBytesRX(res.Trace.HTTP.CallStats.InputBytes)
 			traceSt.addAPIBytesTX(res.Trace.HTTP.CallStats.OutputBytes)
 		}
+		if res.Trace.HTTP.RespInfo.StatusCode >= 500 {
+			traceSt.addAPIErrors(1)
+		}
 		m.apiStatsMap[res.Trace.FuncName] = traceSt
 	}
 
-	table.SetHeader([]string{"API", "CALLS", "RX", "TX"})
+	table.SetHeader([]string{"API", "CALLS", "ERRORS", "RX", "TX"})
```

Comment:
I think can be written 
```suggestion
	table.SetHeader([]string{"API", "CALLS", "ERRORS(5xx)", "RX", "TX"})
```

Reply:
@kannappanr  suggested to consider 499 as error , so not renaming `"ERRORS"` ->  `"ERRORS(5xx)"`